community-connectors/meetup/test/Connector.test.js

```diff
@@ -0,0 +1,716 @@
+    {values: ['event 1 name', '20180223', 1, 12, 'fakeLink', 10, '03:00']},
+    {values: ['event 1 name', '20180223', 1, 12, 'fakeLink', 0, '03:00']},
+    {values: ['event 2 name', '20180622', 0, 10, 'fakeLink2', 25, '02:00']}
+  ];
+
+  expect(actual).toEqual(expected);
+});
+
+test('requestEventsData does the right things.', () => {
+  const response = [
+    {
+      'fee': {
+        'accepts': 'paypal',
+        'amount': 10,
+        'currency': 'USD',
+        'description': 'per person',
+        'label': 'price',
+        'required': true
+      },
+      'link': 'https://www.meetup.com/Igniter/events/244824517/',
+      'yes_rsvp_count': 12,
+      'waitlist_count': 0,
+      'time': 1519354800000
+    },
+    {
+      'fee': {
+        'accepts': 'paypal',
+        'amount': 25,
+        'currency': 'USD',
+        'description': 'per person',
+        'label': 'price',
+        'required': true
+      },
+      'link': 'https://www.meetup.com/Igniter/events/244743694/',
+      'yes_rsvp_count': 10,
+      'waitlist_count': 0,
+      'time': 1529632800000
+    }
+  ];
+
+  const request = buildFakeRequest({
+    apiType: Connector.API_TYPE_EVENTS,
+    fields: [
+      {name: 'event_date'}, {name: 'waitlist_count'}, {name: 'yes_rsvp_count'},
+      {name: 'link'}, {name: 'fee'}, {name: 'event_time'}
+    ]
+  });
+  urlFetchMock.mockImplementation(() => JSON.stringify(response));
+  c.getFetchOptions = () => ({});
+  const url = c.buildURL(request);
+  const actual = c.requestEventsData(request, url);
+  expect(actual).toEqual(response);
+});
+
+test('errors are handled in getCachedData', () => {
+  c.cache = 'not null';
+  c.getFromCachePaginated = jest.fn();
+  c.getCachedData();
+  expect(errorMock.mock.calls[0]).toEqual([
+    'Unable to get data from meetup.com', true
+  ]);
+});
+
+test('paginatedResult test', () => {
+
+  var counter = 0;
+  const json = ['my json', 'response'];
+  c.getCachedData = jest.fn(() => {
+    if (counter < 2) {
+      counter++;
+      return {nextLink: 'my next link', json};
+    } else {
+      return {nextLink: undefined, json};
+    }
+  });
+
+  const actual = c.paginatedResult(undefined, undefined);
+  expect(actual).toEqual(json.concat(json).concat(json));
+});
+
+test('getFetchOptions test', () => {
+
+  const actual = c.getFetchOptions();
+  expect(actual).toEqual(
+      {'headers': {'Authorization': 'Bearer accesstoken'}});
+});
+
+test('requestGeneralInfo test', () => {
+  urlFetchMock.mockImplementation(() => JSON.stringify({}));
+  const actual = c.requestGeneralInfo();
+  expect(actual).toEqual({});
+});
+
+test('getConfig test', () => {
+  const actual = c.getConfig();
+  expect(actual.configParams).toBeTruthy();
+});
+
+test('getAuthType test', () => {
+  const actual = c.getAuthType();
+  expect(actual.type).toBeTruthy();
+});
+
+test('isAdminUser test', () => {
+  const actual = c.isAdminUser();
+  expect(actual).toBe(false);
+});
+
+test('logAndExecute test1', () => {
```

Comment:
Should the name be more descriptive than test1? (here and below)

Reply:
done.